src/libraries/pretest.proj

```diff
@@ -105,15 +104,15 @@
       <FrameworkListRootAttributes Include="FrameworkName" Value="$(SharedFrameworkName)" />
     </ItemGroup>
 
-    <CreateFrameworkListFile Files="@(_runtimePackLibFiles);@(_runtimePackNativeFiles)"
-                             TargetFile="$(_frameworkListFile)"
+    <CreateFrameworkListFile Files="@(_refPackLibFile)"
+                             TargetFile="$(MicrosoftNetCoreAppRefPackDataDir)FrameworkList.xml"
                              TargetFilePrefixes="ref/;runtimes/"
```

Comment:
I'm not familiar with the CreateFrameworkListFile task but is this parameter correct? I ask since we are passing runtimes to it for the Targeting pack which doesn't make much sense.

Reply:
https://microsofteur-my.sharepoint.com/:u:/g/personal/vihofer_microsoft_com/ERH6vPl0VeBHsUwTbAB5PwsBYbzrlPM0qFMrivBSxCXWhg?e=SWVTuG